### ISSUE TITLE
Multi-language support

### ISSUE THREAD

**Issue by ViBE-HU** [andrebrait/prime-indicator#8]
Please add localization support.

**Comment by andrebrait** [comment]
Will do! It's on my plans. I need people to translate, though. I have, at this time, Italian, Brazilian Portuguese and English.

If you'd like to contribute or you know people who can, please point them to me.

**Comment by ViBE-HU** [comment]
Absolutely you can count on me. I already translated the original tool to Hungarian just never published.

**Comment by jsalatas** [comment]
I can contribute greek translations. Just let me know :)

**Comment by andrebrait** [comment]
@ViBE-HU @jsalatas I would appreciate! Can you add the translations to the JSON file in the /usr/lib/prime-indicator folder?

**Comment by jsalatas** [comment]
Attached (as txt as github doesn't allow me to attach json). I guess there might be some issues in the translation due to string concatenation, but I need to see these in action before I can tell for sure 

[prime-indicator.txt](https://github.com/andrebrait/prime-indicator/files/545999/prime-indicator.txt)

**Comment by ViBE-HU** [comment]
@andrebrait not sure how to fork but I think I did it and made a commit (for the very first time lol). Notice me if I did it wrong.

**Comment by andrebrait** [comment]
@ViBE-HU You did it right! I jist merged your pull request! Thanks a lot!

**Comment by andrebrait** [comment]
@jsalatas the user @mugna91 changed some parts of the translations file. Would you mind checking if your translations are still correct when comparing to the latest version of the english strings? Thanks a lot!

**Comment by jsalatas** [comment]
Will do it later. I'll create a pull request for it :)

**Comment by andrebrait** [comment]
Two messages were left out when the original file was created. @mugna91 added them a few hours ago. Can you guys please add the translations for those, when you find some spare time to do it? Thanks a lot!

**Comment by ViBE-HU** [comment]
@andrebrait I noticed this but I thought maybe you will remove these from the code for some reasons. The Logout button still missing, right?

**Comment by andrebrait** [comment]
I haven't been having much time to review these changes beyond a quick look in the pull request before merging. If the Logout button is still missing, we better add it because it appears a lot.

As for those messages, I don't plan to remove them unless there's a standardized way of logging out, and I don't think there is. So, since I don't plan to support every DE out there, it will always be possible for someone to be running an unsupported DE.

**Comment by ViBE-HU** [comment]
@andrebrait Of course. I just wanted to say that the string is missing from the json while it's in the code so you may want to add it also to the text files.

**Comment by mugna91** [comment]
@ViBE-HU You're right. Done a PR.

**Comment by andrebrait** [comment]
Thanks, @mugna91!

**Comment by gabrielssilva** [comment]
@andrebrait do you still need help with pt-br?
I see you have pt-pt translations, and the messages are pretty much the same. But it is not complete, and I can send a patch anyway.

**Comment by andrebrait** [comment]
@gabrielssilva I'm brazilian, so I don't need help with the translation itself. However, I have been quite short in spare time lately, so it would be nice if you could send the patch :)

**Comment by gabrielssilva** [comment]
It's a pretty simple patch, but I hope it helps (#17) :)

**Comment by andrebrait** [comment]
It sure does! Thanks a lot! Merging now!

**Comment by ViBE-HU** [comment]
I was fairly inactive but I wanted to notice that the translation does not appears on Ubuntu. It's still use English somehow on my system. I also tested on a freshly installed laptop, but same happens.

**Comment by andrebrait** [comment]
Hey there, Vibe. Yeah, it has not been implemented yet. I have been busy
lately. Sorry about that :/

Em 19 de abr de 2017 3:06 PM, "ViBE" <notifications@github.com> escreveu:
